### ISSUE TITLE
Macro drag creation not working anymore

### ISSUE THREAD

**Issue by LinkDragmire** [AnthonyMonette/TheWitcherTRPG#286]
Since the last update, dragging a weapon or a spell into the hotbar no longer creates a working macro to use it for an attack, it states the error "You can only create macro buttons for owned items" and creates a macro to show the item instead.

**Comment by Melkkart** [comment]
Hello, it works correctly for me, but, I have to drag from the icon, if I drag from the name, I get the same error as you, try to drag from the icon to the bar.

**Comment by Melkkart** [comment]
ok, try this.
![Sin título](https://user-images.githubusercontent.com/123239719/222769550-ac376f36-959b-418c-9b07-3c29e8c43531.jpg)

Create This macro, only need change ID of object and actor

**Comment by LinkDragmire** [comment]
this way for creating works. my players could create macros like before too. Seems to be an GM only issue right now.

**Comment by AnthonyMonette** [comment]
I tried on my side and it seems to work fine with the latest version.  
@LinkDragmire could you try and confirm if this is still an issue, Thanks!

**Comment by LinkDragmire** [comment]
will try next week, when I'm able to access my setup

**Comment by LinkDragmire** [comment]
Ok, I just tested with the following result:

It seems to be a problem with the default ownership.
This was set to limited in my world for the player tokens. So my "GM"-Account could see the token and handle it, but the permission was limited. Crreating a macro from a PC didn't work. I tried setting the ownership to default value None. Now It's working for the GM-Account. I will set the ownership for the other players to limited for each on their own. 

Maybe It's an issue, which could be ignored. Maybe it could be handled, that  GM-Accounts always count as owning Tokens.

For me, I found a solution, could be closed. But i would participate in finding a solution, if someone wishes to do it.

@AnthonyMonette what's your thoughts about it?
